gcs/src/fla.jsx

```diff
@@ -238,12 +238,15 @@ export default function FLA() {
         setLoadingFile(false)
       }
     }
+    if(file == null){
```

Comment:
preferably, the code style should go `if (file == null) {` with the spaces

Reply:
okay